ci/requirements-2.7_LOCALE.run

```diff
@@ -4,6 +4,7 @@ numpy=1.8.2
 xlwt=0.7.5
 openpyxl=1.6.2
 xlsxwriter=0.4.6
+cssdecl
```

Comment:
don't add this to all of these
just 2.7 and 3.6 is enough
u can add the os x build if u want
windows prob 2 (only need to add to 1)